index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <section id="nav">
```

Comment:
- [ ] Please for semantics purpose, kindly use the `header `tag in place of the `section `tag. The [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents introductory content, typically a group of introductory or navigational aids. 👍